src/test/java/lotto/WinningNumbersTest.java

```diff
@@ -0,0 +1,35 @@
+package lotto;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class WinningNumbersTest {
+
+  @DisplayName("당첨 번호와 보너스 번호가 중복되면 예외가 발생한다")
+  @Test
+  void duplicateWinningNumbers() {
+    Lotto winningNumbers = new Lotto("1,2,3,4,5,6");
+    LottoNumber bonus = new LottoNumber(6);
+    assertThatIllegalArgumentException().isThrownBy(() -> new WinningNumbers(winningNumbers, bonus))
+        .withMessageContaining("중복");
+  }
+
+  @Test
+  void containsWinningNumber() {
+    WinningNumbers wn = new WinningNumbers(new Lotto("1,2,3,4,5,6"), new LottoNumber(7));
+    assertThat(wn.contains(new LottoNumber(3))).isTrue();
+    assertThat(wn.contains(new LottoNumber(7))).isFalse();
```

Comment:
```suggestion
    WinningNumbers wn = new WinningNumbers(new Lotto("1,2,3,4,5,6"), 7);
    assertThat(wn.contains(7)).isTrue();
    assertThat(wn.contains(7)).isFalse();
```
WinningNumbers에 부 생성자 추가
contains()에 int도 인자를 받을 수 있도록 메서드를 오버로딩하면 어떨까?

이 두 가지를 고려하면서 구현할 경우 WinningNumbers와 메서드를 사용하는 개발자 입장에서 편리성을 얻을 수 있지 않을까?